src/quantum/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.20.0
+++++++
+* [2023-12-13] Version intended to work with QDK version 0.29.2
```

Comment:
Where does this version come from?

Reply:
This is the latest version of [azure-quantum-python](https://pypi.org/project/azure-quantum/0.29.2/) aka QDK that we released last week. I thought it would be natural to continue the trend and specify which QDK version this extension version would work with.